app/views/users/passwords/edit.html.erb

```diff
@@ -0,0 +1,25 @@
+<h2>Change your password</h2>
+
+<%= form_with(model: resource, url: password_path(resource_name), method: :put, data: { turbo: false }) do |f| %>
+  <%= render "users/shared/error_messages", resource: resource %>
+  <%= f.hidden_field :reset_password_token %>
+
+  <div class="field">
+    <%= f.label :password, "New password" %><br />
```

Comment:
Using `<br />` is not ideal. We could later add more styles to this just so we don't promote those bad practices.

Reply:
Yeah I agree, these are the views that Devise generated, but I can fix them

Reply:
Agree with @brunvez

Reply:
We can make a ticket to "Fix styles" or something. This PR is already big and that way we can split the work